.storybook-s2/docs/Migrating.jsx

```diff
@@ -0,0 +1,775 @@
+import { style } from '../../packages/@react-spectrum/s2/style/spectrum-theme' with {type: 'macro'};
+import {Link} from '@react-spectrum/s2';
+import {P, Code, Pre, H3, H2} from './typography';
+
+export function Migrating() {
+  return (
+    <div className={style({maxWidth: 'lg', marginX: 'auto', fontFamily: 'sans'})}>
+      <div className={style({marginX: 48})}>
+        <h1 className={style({fontSize: 'heading-2xl', color: 'heading', marginBottom: 48})}>
+          Migrating to Spectrum 2
+        </h1>
+        <P>An automated upgrade assistant is available by running the following command in the project you want to upgrade:</P>
+        <Pre>npx @react-spectrum/codemods s1-to-s2</Pre>
+        <details>
```

Comment:
I assume there's other sections coming?
Otherwise, I don't see a need to hide this. Instead, I'd introduce it with a little sentence along these lines

"If running the above script was incomplete.... see below for descriptions of how to manually migrate specific props or patterns" or "Other decisions may be needed for migrations that do not fall into what we can do with a codemod script. See below... blah blah blah"

Reply:
Sounds good, ended up removing the collapsing behavior, and rewording it.